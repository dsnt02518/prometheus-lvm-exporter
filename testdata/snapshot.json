{
  "report": [
    {
      "vg": [
        {}
      ],
      "pv": [
        {}
      ],
      "lv": [
        {
          "lv_uuid": "b3Wegf-3TOd-I5YS-ll0X-RZgI-QxAu-qv0fbc",
          "lv_name": "lv_home",
          "lv_ancestors": "",
          "lv_descendants": "lv_home_snapshot",
          "lv_full_ancestors": "",
          "lv_full_descendants": "lv_home_snapshot"
        },
        {
          "lv_uuid": "Il5cUA-BDjk-T5ZB-vTdr-EsJd-6bFa-zjuhgB",
          "lv_name": "lv_root",
          "lv_ancestors": "",
          "lv_descendants": "",
          "lv_full_ancestors": "",
          "lv_full_descendants": ""
        },
        {
          "lv_uuid": "FNkhIS-Vozm-bmE7-dmHD-vjOM-103g-irxDIb",
          "lv_name": "lv_docker",
          "lv_ancestors": "",
          "lv_descendants": "",
          "lv_full_ancestors": "",
          "lv_full_descendants": ""
        },
        {
          "lv_uuid": "PucjXb-iwvM-Px3b-tfO8-xy43-F8B4-2uEyc0",
          "lv_name": "lv_libvirt_images",
          "lv_ancestors": "",
          "lv_descendants": "",
          "lv_full_ancestors": "",
          "lv_full_descendants": ""
        },
        {
          "lv_uuid": "VBJ3Ct-yDJ3-JPGS-Zyew-tCO5-7AvK-INt3g6",
          "lv_name": "lv_csmeta",
          "lv_ancestors": "",
          "lv_descendants": "",
          "lv_full_ancestors": "",
          "lv_full_descendants": ""
        },
        {
          "lv_uuid": "zqJxaj-K6iF-f1NV-4G35-8Pg7-hhpn-p77SMP",
          "lv_name": "lv_home_snapshot",
          "lv_ancestors": "lv_home",
          "lv_descendants": "",
          "lv_full_ancestors": "lv_home",
          "lv_full_descendants": ""
        }
      ],
      "pvseg": [
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": "FNkhIS-Vozm-bmE7-dmHD-vjOM-103g-irxDIb"
        },
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": "PucjXb-iwvM-Px3b-tfO8-xy43-F8B4-2uEyc0"
        },
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": "b3Wegf-3TOd-I5YS-ll0X-RZgI-QxAu-qv0fbc"
        },
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": "Il5cUA-BDjk-T5ZB-vTdr-EsJd-6bFa-zjuhgB"
        },
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": "FNkhIS-Vozm-bmE7-dmHD-vjOM-103g-irxDIb"
        },
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": "zqJxaj-K6iF-f1NV-4G35-8Pg7-hhpn-p77SMP"
        },
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": ""
        },
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": "PucjXb-iwvM-Px3b-tfO8-xy43-F8B4-2uEyc0"
        },
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": "VBJ3Ct-yDJ3-JPGS-Zyew-tCO5-7AvK-INt3g6"
        },
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": ""
        },
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": "b3Wegf-3TOd-I5YS-ll0X-RZgI-QxAu-qv0fbc"
        },
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": ""
        },
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": "b3Wegf-3TOd-I5YS-ll0X-RZgI-QxAu-qv0fbc"
        },
        {
          "pv_uuid": "DobPwo-HVJ3-hjKh-m4q7-y4e6-bmHi-2avBcN",
          "lv_uuid": "PucjXb-iwvM-Px3b-tfO8-xy43-F8B4-2uEyc0"
        }
      ],
      "seg": [
        {
          "lv_uuid": "b3Wegf-3TOd-I5YS-ll0X-RZgI-QxAu-qv0fbc"
        },
        {
          "lv_uuid": "b3Wegf-3TOd-I5YS-ll0X-RZgI-QxAu-qv0fbc"
        },
        {
          "lv_uuid": "b3Wegf-3TOd-I5YS-ll0X-RZgI-QxAu-qv0fbc"
        },
        {
          "lv_uuid": "Il5cUA-BDjk-T5ZB-vTdr-EsJd-6bFa-zjuhgB"
        },
        {
          "lv_uuid": "FNkhIS-Vozm-bmE7-dmHD-vjOM-103g-irxDIb"
        },
        {
          "lv_uuid": "FNkhIS-Vozm-bmE7-dmHD-vjOM-103g-irxDIb"
        },
        {
          "lv_uuid": "PucjXb-iwvM-Px3b-tfO8-xy43-F8B4-2uEyc0"
        },
        {
          "lv_uuid": "PucjXb-iwvM-Px3b-tfO8-xy43-F8B4-2uEyc0"
        },
        {
          "lv_uuid": "PucjXb-iwvM-Px3b-tfO8-xy43-F8B4-2uEyc0"
        },
        {
          "lv_uuid": "VBJ3Ct-yDJ3-JPGS-Zyew-tCO5-7AvK-INt3g6"
        },
        {
          "lv_uuid": "zqJxaj-K6iF-f1NV-4G35-8Pg7-hhpn-p77SMP"
        }
      ]
    }
  ]
}
