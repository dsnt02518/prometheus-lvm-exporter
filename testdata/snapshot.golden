# HELP lvm_lv_info 
# TYPE lvm_lv_info gauge
lvm_lv_info{lv_active="",lv_allocation_policy="",lv_ancestors="",lv_attr="",lv_convert_lv="",lv_convert_lv_uuid="",lv_data_lv="",lv_data_lv_uuid="",lv_descendants="",lv_dm_path="",lv_full_ancestors="",lv_full_descendants="",lv_full_name="",lv_health_status="",lv_host="",lv_kernel_cache_policy="",lv_kernel_cache_settings="",lv_kernel_discards="",lv_kernel_metadata_format="",lv_kernel_read_ahead_bytes="",lv_layout="",lv_metadata_lv="",lv_metadata_lv_uuid="",lv_mirror_log="",lv_mirror_log_uuid="",lv_modules="",lv_move_pv="",lv_move_pv_uuid="",lv_name="lv_csmeta",lv_origin="",lv_origin_uuid="",lv_parent="",lv_path="",lv_permissions="",lv_pool_lv="",lv_pool_lv_uuid="",lv_raid_sync_action="",lv_raidintegritymode="",lv_role="",lv_tags="",lv_uuid="VBJ3Ct-yDJ3-JPGS-Zyew-tCO5-7AvK-INt3g6",lv_vdo_operating_mode="",lv_when_full=""} 1
lvm_lv_info{lv_active="",lv_allocation_policy="",lv_ancestors="",lv_attr="",lv_convert_lv="",lv_convert_lv_uuid="",lv_data_lv="",lv_data_lv_uuid="",lv_descendants="",lv_dm_path="",lv_full_ancestors="",lv_full_descendants="",lv_full_name="",lv_health_status="",lv_host="",lv_kernel_cache_policy="",lv_kernel_cache_settings="",lv_kernel_discards="",lv_kernel_metadata_format="",lv_kernel_read_ahead_bytes="",lv_layout="",lv_metadata_lv="",lv_metadata_lv_uuid="",lv_mirror_log="",lv_mirror_log_uuid="",lv_modules="",lv_move_pv="",lv_move_pv_uuid="",lv_name="lv_docker",lv_origin="",lv_origin_uuid="",lv_parent="",lv_path="",lv_permissions="",lv_pool_lv="",lv_pool_lv_uuid="",lv_raid_sync_action="",lv_raidintegritymode="",lv_role="",lv_tags="",lv_uuid="FNkhIS-Vozm-bmE7-dmHD-vjOM-103g-irxDIb",lv_vdo_operating_mode="",lv_when_full=""} 1
lvm_lv_info{lv_active="",lv_allocation_policy="",lv_ancestors="",lv_attr="",lv_convert_lv="",lv_convert_lv_uuid="",lv_data_lv="",lv_data_lv_uuid="",lv_descendants="",lv_dm_path="",lv_full_ancestors="",lv_full_descendants="",lv_full_name="",lv_health_status="",lv_host="",lv_kernel_cache_policy="",lv_kernel_cache_settings="",lv_kernel_discards="",lv_kernel_metadata_format="",lv_kernel_read_ahead_bytes="",lv_layout="",lv_metadata_lv="",lv_metadata_lv_uuid="",lv_mirror_log="",lv_mirror_log_uuid="",lv_modules="",lv_move_pv="",lv_move_pv_uuid="",lv_name="lv_libvirt_images",lv_origin="",lv_origin_uuid="",lv_parent="",lv_path="",lv_permissions="",lv_pool_lv="",lv_pool_lv_uuid="",lv_raid_sync_action="",lv_raidintegritymode="",lv_role="",lv_tags="",lv_uuid="PucjXb-iwvM-Px3b-tfO8-xy43-F8B4-2uEyc0",lv_vdo_operating_mode="",lv_when_full=""} 1
lvm_lv_info{lv_active="",lv_allocation_policy="",lv_ancestors="",lv_attr="",lv_convert_lv="",lv_convert_lv_uuid="",lv_data_lv="",lv_data_lv_uuid="",lv_descendants="",lv_dm_path="",lv_full_ancestors="",lv_full_descendants="",lv_full_name="",lv_health_status="",lv_host="",lv_kernel_cache_policy="",lv_kernel_cache_settings="",lv_kernel_discards="",lv_kernel_metadata_format="",lv_kernel_read_ahead_bytes="",lv_layout="",lv_metadata_lv="",lv_metadata_lv_uuid="",lv_mirror_log="",lv_mirror_log_uuid="",lv_modules="",lv_move_pv="",lv_move_pv_uuid="",lv_name="lv_root",lv_origin="",lv_origin_uuid="",lv_parent="",lv_path="",lv_permissions="",lv_pool_lv="",lv_pool_lv_uuid="",lv_raid_sync_action="",lv_raidintegritymode="",lv_role="",lv_tags="",lv_uuid="Il5cUA-BDjk-T5ZB-vTdr-EsJd-6bFa-zjuhgB",lv_vdo_operating_mode="",lv_when_full=""} 1
lvm_lv_info{lv_active="",lv_allocation_policy="",lv_ancestors="",lv_attr="",lv_convert_lv="",lv_convert_lv_uuid="",lv_data_lv="",lv_data_lv_uuid="",lv_descendants="lv_home_snapshot",lv_dm_path="",lv_full_ancestors="",lv_full_descendants="lv_home_snapshot",lv_full_name="",lv_health_status="",lv_host="",lv_kernel_cache_policy="",lv_kernel_cache_settings="",lv_kernel_discards="",lv_kernel_metadata_format="",lv_kernel_read_ahead_bytes="",lv_layout="",lv_metadata_lv="",lv_metadata_lv_uuid="",lv_mirror_log="",lv_mirror_log_uuid="",lv_modules="",lv_move_pv="",lv_move_pv_uuid="",lv_name="lv_home",lv_origin="",lv_origin_uuid="",lv_parent="",lv_path="",lv_permissions="",lv_pool_lv="",lv_pool_lv_uuid="",lv_raid_sync_action="",lv_raidintegritymode="",lv_role="",lv_tags="",lv_uuid="b3Wegf-3TOd-I5YS-ll0X-RZgI-QxAu-qv0fbc",lv_vdo_operating_mode="",lv_when_full=""} 1
lvm_lv_info{lv_active="",lv_allocation_policy="",lv_ancestors="lv_home",lv_attr="",lv_convert_lv="",lv_convert_lv_uuid="",lv_data_lv="",lv_data_lv_uuid="",lv_descendants="",lv_dm_path="",lv_full_ancestors="lv_home",lv_full_descendants="",lv_full_name="",lv_health_status="",lv_host="",lv_kernel_cache_policy="",lv_kernel_cache_settings="",lv_kernel_discards="",lv_kernel_metadata_format="",lv_kernel_read_ahead_bytes="",lv_layout="",lv_metadata_lv="",lv_metadata_lv_uuid="",lv_mirror_log="",lv_mirror_log_uuid="",lv_modules="",lv_move_pv="",lv_move_pv_uuid="",lv_name="lv_home_snapshot",lv_origin="",lv_origin_uuid="",lv_parent="",lv_path="",lv_permissions="",lv_pool_lv="",lv_pool_lv_uuid="",lv_raid_sync_action="",lv_raidintegritymode="",lv_role="",lv_tags="",lv_uuid="zqJxaj-K6iF-f1NV-4G35-8Pg7-hhpn-p77SMP",lv_vdo_operating_mode="",lv_when_full=""} 1
# HELP lvm_pv_info 
# TYPE lvm_pv_info gauge
lvm_pv_info{pv_attr="",pv_fmt="",pv_name="",pv_tags="",pv_uuid=""} 1
# HELP lvm_unknown_field_count Fields reported by LVM not recognized by exporter
# TYPE lvm_unknown_field_count gauge
lvm_unknown_field_count{details="",group="lv"} 0
lvm_unknown_field_count{details="",group="pv"} 0
lvm_unknown_field_count{details="",group="vg"} 0
# HELP lvm_up Whether scrape was successful
# TYPE lvm_up gauge
lvm_up{status=""} 1
# HELP lvm_vg_info 
# TYPE lvm_vg_info gauge
lvm_vg_info{vg_allocation_policy="",vg_attr="",vg_fmt="",vg_lock_args="",vg_lock_type="",vg_name="",vg_permissions="",vg_systemid="",vg_tags="",vg_uuid=""} 1
